packages/slice-machine/src/domain/fields.ts

```diff
@@ -34,12 +34,44 @@ export interface GroupField extends BaseField {
 
 export type Field = NestableField | UIDField | GroupField;
 
+export type CustomFieldType =
+  | FieldType
+  | "LinkToMedia"
+  | "ContentRelationship"
+  | "NestedGroup";
+
+export const fieldLabels: Record<CustomFieldType, string> = {
+  Text: "Text",
+  StructuredText: "Rich Text",
```

Comment:
```suggestion
  StructuredText: "Rich text",
```
i thought we didn't want to use title case anymore, same for the rest

Reply:
I copied these from the add field dropdown and left them as they were 😕

<img width="623" height="464" alt="Screenshot 2025-08-06 at 16 03 44" src="https://github.com/user-attachments/assets/9a5f8598-eb43-45c1-b960-c08aba6d9079" />